e2e-tests/IIoTPlatform-E2E-Tests/Standalone/PublishNodesFromFileTestTheory.cs

```diff
@@ -37,25 +37,58 @@ public class PublishNodesFromFileTestTheory {
         }
 
         [Fact, PriorityOrder(2)]
-        public async Task PublishFromPublishedNodesFile() {
+        public async Task Test_SwitchToStandaloneMode() {
+            var simulatedOpcServer = await TestHelper.GetSimulatedOpcUaNodesAsync(_context);
+            var numberOfNodes = 1;
+            //Create and save a published_nodes.json file with numberOfNodes nodes
+            TestHelper.SavePublishedNodesFile(simulatedOpcServer[simulatedOpcServer.Keys.First()], numberOfNodes, _context);
+            _output.WriteLine("Saved published_nodes.json file");
+
+            TestHelper.SwitchToStandaloneMode(_context);
+            TestHelper.LoadPublishedNodesFile(_context.PublishedNodesFileInternalFolder, TestConstants.PublishedNodesFolder + "/" + TestConstants.PublisherPublishedNodesFile, _context);
+            _output.WriteLine("Switched to standalone mode and loaded published_nodes.json file");
+        }
+
+        [Fact, PriorityOrder(3)]
+        public async Task Test_CreateEdgeBaseDeployment_Expect_Success() {
+            var deploy = new IoTHubEdgeBaseDeployment(_context);
```

Comment:
>IoTHubEdgeBaseDeployment [](start = 29, length = 24)

instead of creating objects directly, add IoTHubEdgeBaseDeployment as property to context, and access it only via interface

Reply:
done